test/asv.linux.conf.json

```diff
@@ -34,20 +34,20 @@
         "python -c \"import shutil; shutil.rmtree('{build_dir}/qiskit', True)\"",
         "python -c \"import shutil; shutil.rmtree('{build_dir}/qiskit_aer.egg-info', True)\"",
         "pip install git+https://github.com/Qiskit/qiskit-terra",
-        "pip install git+https://github.com/Qiskit/qiskit-aqua",
+        "pip install qiskit_nature",
         "python -mpip install {wheel_file}"
     ],
     "uninstall_command": [
         "return-code=any python -mpip uninstall -y qiskit-terra",
-        "return-code=any python -mpip uninstall -y qiskit-aqua",
+        "return-code=any python -mpip uninstall -y qiskit_nature",
         "return-code=any python -mpip uninstall -y {project}"
```

Comment:
Just for consistency, it may be easier as `qiskit-nature` (with a hyphen, not an underscore).